pkg/model/resources/nodeup.go

```diff
@@ -149,12 +150,13 @@ function download-release() {
 
 /bin/systemd-machine-id-setup || echo "failed to set up ensure machine-id configured"
 
-{{ if RestartJournald -}}
-# DO cloud provider has /etc/machine-id baked in the image and 
-# systemd-machine-id-setup creates a new id and journalctld should be flushed
-# to use the new machine-id
-systemctl restart systemd-journald
-{{- end }}
+if [[ "${CLOUD_PROVIDER}" = "digitalocean" ]]; then
```

Comment:
Maybe still do this at template level like before? No need to have it for all clouds, in bash.

Reply:
Or maybe always restart `systemd-journald`, without any check?

Reply:
I will probably keep this check since this issue was only pertaining to Digital Ocean. Though, restarting journald should not cause any issues as such..

Reply:
Also, please remove the comment or make it very short, without echo.
The size for user-data is quite small. We don't want to add extra unneeded chars.

Reply:
Sure, done.